apiblueprint-request.mustache

```diff
@@ -1,4 +1,4 @@
-# {{{ method }}} {{{ path }}}
+{{{ level }}} {{{ name }}} [{{{ method }}} {{{ path }}}]
```

Comment:
We should only use the API Blueprint syntax with a name if there is a name set:

```suggestion
{{#name}}
{{{ level }}} {{{ name }}} [{{{ method }}} {{{ path }}}]
{{/name}}
{{^name}}
{{{ level }}} {{{ method }}} {{{ path }}}
{{/name}}
```
Otherwise it renders as such:

![Screenshot 2020-03-26 at 17 31 53](https://user-images.githubusercontent.com/44164/77677509-b53a8200-6f87-11ea-8cd1-964620e96d02.png)